### PR TITLE
✨ Migrate non-condition resources

### LINKED ISSUE #3: Migrate remaining resources
Migrate remaining resources. Whether they will be migrated as they are or with modifications needs to be discussed (See issues here: https://github.com/ncpi-fhir/ncpi-model-forge/issues): 

- Re-profile phenotypes as Conditions
- (Optional) re-profile pedigree (currently family relationship) against the proposal of GA4GH Pedigree Working Group: https://github.com/ga4gh-cp/pedigree/blob/2021-01/model.md

### PR DESCRIPTION
**Pull request by liberaliscomputing**
This PR migrates non-condition resources from https://github.com/ncpi-fhir/ncpi-model-forge. This PR closes #3.

### CODE REVIEW

File: input/fsh/Instance-DRSDocumentReference.fsh [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-* url = "drs://example.com/ga4gh/drs/v1/objects/0f8c27b9-3300-4249-bb28-f49ffb80e277"
-
-Instance: drs-document-reference-example
+Instance: drs-document-reference-example-1
 InstanceOf: DRSDocumentReference
-Description: "An example representation of a DRSDocumentReference"
+Description: "An example representation of a DRS Document Reference"
 * status = #current
-* content.attachment = drs-attachment-example
+* content.attachment = drs-attachment-example-1
```
Comment (bwalsh) [inline input/fsh/Instance-DRSDocumentReference.fsh:9]: Looks good.  Separating attachment into its own file makes sense

**Review by bwalsh — DISMISSED** [review]
Looks good

File: input/fsh/Instance-Disease.fsh [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-* recordedDate.extension[0].extension[1].valueString = "birthDate"
-* recordedDate.extension[0].extension[2].url = "relationship"
-* recordedDate.extension[0].extension[2].valueCode = #after
-* recordedDate.extension[0].extension[3].url = "offset"
-* recordedDate.extension[0].extension[3].valueDuration = 25 'days'
-* recordedDate.extension[0].extension[3].valueDuration.unit = "d"
\ No newline at end of file
+* subject = Reference(patient-example-1)
+* recordedDate.extension.url = $cqf-relativeDateTime
+* recordedDate.extension.extension[0].url = "target"
```
Comment (torstees) [inline input/fsh/Instance-Disease.fsh:18]: nice! I wasn't aware you could drop the index for the singular extensions.

File: input/fsh/Instance-Specimen.fsh [hunk 3]
```diff
@@ -1,22 +1,38 @@
...
-* identifier[1].value = "Specimen|SD_BHJXBDQK|571312"
-* type.coding = SpecimenTypeCodeSystem#TISS "Tissue"
+Usage: #example
+Description: "Example of the representation of Specimen."
+* type = $v2-0487#TISS "Tissue"
 * type.text = "Solid Tissue"
 * subject = Reference(patient-example-1)
-* receivedTime = "2019-06-15"
-* collection.collectedDateTime = "2019-06-15"
+* receivedTime.extension.url = $cqf-relativeDateTime
```
Comment (torstees) [inline input/fsh/Instance-Specimen.fsh:12]: Good call on the alias here. I hadn't even thought about it appearing multiple times.

**Review by torstees — APPROVED** [review]
Looks good to me.

File: input/fsh/Instance-Disease.fsh (hunk 2 above)
Comment (liberaliscomputing) [reply input/fsh/Instance-Disease.fsh:18]: Oh yeah, when there is a single element, indexing is not necessary!

**Review by bwalsh — APPROVED** [review]
Looks good.  I compiled the fsh files and generate the IG locally

**Pull request merged**